webapp.yml

```diff
@@ -0,0 +1,71 @@
+# Description
+# ===========
+# This playbook create an Azure Linux Web app with Java container Tomcat 8.5,
+# then build a java hello world app, then ftp the war file to the Web app.
```

Comment:
build -> builds
java - I think should be uppercase
too many thens

Reply:
fixed